tests/unit/algos/list/delimiterMaster.spec.js

```diff
@@ -46,6 +47,111 @@ function getListingKey(key, vFormat) {
     return assert.fail(`bad vFormat ${vFormat}`);
 }
 
+const MAX_STREAK_LENGTH = 100
```

Comment:
I though that the linter would complain on this, maybe the rules have been updated? Anyway I think it's a good practice to end all instructions with a semicolon.

```suggestion
const MAX_STREAK_LENGTH = 100;
```